theme_5/src/styles/vendor/header.scss

```diff
@@ -0,0 +1,112 @@
+
+/*================ HEADER STYLES ================*/
+
+.main-header {
+  background: $white-solid;
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: space-around;
+  opacity: .9;
+  padding: 0 10px;
+  width: 100%;
+}
+
+.header-top {
+  border-bottom: 1px solid $cerise-red;
+  display: flex;
+}
+
+.search-container {
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  width: 60%;
+
+  .search-input {
+    border: none;
+    box-shadow: none;
+    color: $black-solid;
+    font-family: $merriweather-font;
+    font-size: 17px;
+    margin-left: 10px;
+    outline: none;
+    padding-left: 0;
+    text-transform: uppercase;
+  }
+}
+
+.nav-container {
+  align-items: center;
+  display: flex;
+  width: 40%;
+}
+
+.secondary-nav {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.option-list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+
+  .option-item {
+    margin: 0 10px;
+    a {
+      color: $black-solid;
+    }
+  }
+
+}
+
+.header-mid {
+  display: flex;
+  justify-content: center;
+}
+
+.logo-img {
+  background-image: url('../assets/logos_png/logo_fiorella.png');
+  height: 82px;
+  transform: scale(.8);
+  width: 250px;
+}
+
+.header-bot {
+  display: flex;
+  justify-content: center;
+}
+
+.collection-list {
+  display: flex;
+  height: 30px;
+  list-style: none;
+
+  .collection-item {
+    border-right: 2px solid $cerise-red;
+    display: flex;
+    flex-direction: row;
+    font-family: $playfair-font;
+    height: 25px;
+    padding: 0 26px;
+    text-transform: uppercase;
+    a {
+      color: $black-solid;
+      &:hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3